saleor/graphql/checkout/tests/test_checkout_lines.py

```diff
@@ -491,6 +491,27 @@ def test_checkout_lines_update_channel_without_shipping_zones(
     assert errors[0]["field"] == "quantity"
 
 
+def test_checkout_lines_update_variant_quantity_over_avability_stock(
+    user_api_client, checkout_with_item
+):
+    checkout = checkout_with_item
+    line = checkout.lines.first()
+    variant_id = graphene.Node.to_global_id("ProductVariant", line.variant.pk)
+    current_stock = line.variant.stocks.first()
+    line.quantity = current_stock.quantity - 1
+    line.save()
+
+    variables = {
+        "token": checkout.token,
+        "lines": [{"variantId": variant_id, "quantity": current_stock.quantity - 1}],
```

Comment:
In this case, you call mutation with the same quantity that is set in the checkout line.